projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -81,9 +98,18 @@ const FirewallHeader = ( { status, hasRequiredPlan } ) => {
 								{ __( 'Active', 'jetpack-protect' ) }
 							</Text>
 							<H3 className={ styles[ 'firewall-heading' ] } mb={ 1 } mt={ 2 }>
-								{ __( 'Automatic firewall is on', 'jetpack-protect' ) }
+								{ hasRequiredPlan
+									? __( 'Automatic firewall is on', 'jetpack-protect' )
+									: __( 'Jetpack firewall is on', 'jetpack-protect' ) }
```

Comment:
Looks like the build is failing on this translation ternary - I believe we need to pass a dummy arg to avoid bad minification here.

Reply:
Classic! Should be resolved via 7dc7ad9a577.